CONTRIBUTING.md

```diff
@@ -57,6 +72,26 @@ You can run test as following:
 ```shell
 make test
 ```
+
+### GitHub Actions local setup using 'act'
```

Comment:
It is very cool idea to use `act` but may we add also a quick guide how to run spark-connect tests manually?

Reply:
Yes, that would be great @SemyonSinchenko. Do you have any samples that I could refer to for writing this guide?

PS: I have been using `act` to test all these test cases for weeks now, but every time I run it, it downloads PySpark every now and then along with all the other libraries, which takes a long time compared to the GH Actions 😄

Reply:
Yep sure, thing will do. I will update the guide with information regarding that.

Reply:
I have updated the `CONTRIBUTING.md` file to add a guide on executing Spark-Connect tests locally.